pset-iterators.js

```diff
@@ -62,6 +62,15 @@ declared with const
 console.log('Problem 1:')
 
 // Add your code below this line
+const presidentsNamedJames = []
```

Comment:
@rvaswani85 nice approach using `forEach()` - using `filter()` would be more efficient here